marti_common_msgs/msg/ServiceHeader.msg

```diff
@@ -0,0 +1,5 @@
+# This message contains all of the data filled in automatically by the service system
+# It is required in each request and response message to be named header
+time stamp
```

Comment:
Does it make sense to have this here (and to require this to be named 'header')?  What if somebody wants to send a request/response that includes a `std_msgs/Header`?

Reply:
I can't forsee a reason someone would want to do it but it would be good to avoid potential conflicts. Do you have any suggestions for the name? 

Also, are you asking if this comment should be here?